website/index.html

```diff
@@ -23,6 +23,11 @@
   	<link rel='preconnect' href='https://fonts.gstatic.com' crossorigin>
   	<link href='https://fonts.googleapis.com/css2?family=Merienda:wght@300;900&family=Poppins:ital,wght@0,300;0,400;0,500;0,600;0,700;0,900;1,400;1,700&family=Hind:wght@300;500;700&display=swap' rel='stylesheet'>
     <link href="https://fonts.googleapis.com/css2?family=Noto+Sans:ital,wdth,wght@0,62.5..100,100..900;1,62.5..100,100..900&display=swap" rel="stylesheet">
+	<link rel="icon" type="image/png" href="/src/icons/favicon-96x96.png" sizes="96x96" />
+	<link rel="icon" type="image/svg+xml" href="/src/icons/favicon.svg" />
+	<link rel="shortcut icon" href="/src/icons/favicon.ico" />
+	<link rel="apple-touch-icon" sizes="180x180" href="/src/icons/apple-touch-icon.png" />
+	<meta name="apple-mobile-web-app-title" content="DiscordBuilders" />
```

Comment:
It seems like the branding PR made its way into this one. You might have to cherry pick it

Reply:
Yea, silly mistake, but this will go away when I squash and merge this PR